pkg/cmd/fetch/fetch.go

```diff
@@ -3337,6 +3338,43 @@ func newCmdWindowsCVRF() *cobra.Command {
 	return cmd
 }
 
+func newCmdWindowsCSAF() *cobra.Command {
+	options := &struct {
+		base
+		concurrency int
+		wait        int
+	}{
+		base: base{
+			dir:   filepath.Join(util.CacheDir(), "fetch", "windows", "csaf"),
+			retry: 3,
+		},
+		concurrency: 5,
+		wait:        1,
+	}
+
+	cmd := &cobra.Command{
+		Use:   "windows-csaf",
+		Short: "Fetch Windows CSAF data source",
+		Example: heredoc.Doc(`
+			$ vuls-data-update fetch windows-csaf
+		`),
+		Args: cobra.NoArgs,
+		RunE: func(_ *cobra.Command, _ []string) error {
+			if err := windowsCSAF.Fetch(windowsCSAF.WithDir(options.dir), windowsCSAF.WithRetry(options.retry)); err != nil {
```

Comment:
Need to pass concurrency and wait ?

Reply:
fixed: https://github.com/MaineK00n/vuls-data-update/compare/862456ae7e9df3ee953a5b2d46b7051dd79dcc3f..9530b96d610663827b3bc6bd5c951d36ff0b069d